project-koku/masu
# Removing expired OCP storage files

Removing expired OCP storage line items as well as cleaning up processed .csv files

**Testing**
1.  Add OCP provider
2.  Ingest OCP data with only usage report (nise) and inspect temp directory
3.  Ingest OCP data (usage and storage).  Using upload service.  Verify that the old .csv is removed and the two report files reamin
4. Set masu date to at least 3 months into the future
5.  Run expired_data endpoint and verify storage line items were removed

**Test Results**
[masu_storage_expired_ut.txt](https://github.com/project-koku/masu/files/2862029/masu_storage_expired_ut.txt)

Closes #347
